Tabloid/Repositories/TagRepository.cs

```diff
@@ -37,6 +38,59 @@ public List<Tag> GetAllTags()
 
                 }
             }
+
+
+        }
+
+
+        public void Add(Tag tag)
+        {
+            using (SqlConnection conn = Connection)
+            {
+                conn.Open();
+                using (SqlCommand cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = @" INSERT Into Tag(Name)
+	                   OUTPUT INSERTED.ID values(@Name)";
+                    cmd.Parameters.AddWithValue("@name", tag.Name);
```

Comment:
Use DbUtils to do this?

Reply:
@joshdbarton  added DbUtils to replace the cmd on the TagRepository.cs